src/test_runner.js

```diff
@@ -563,6 +565,20 @@ class TestRunner {
         });
       }
 
+    } else if (this.enablePassedTestsLogging && !_.isEmpty(passedTests)) {
```

Comment:
Did you mean your passedTests logs will only happen if no test or all tests passed?

Reply:
For tests that pass, we want to log the test steps. 

* If one test pass and one fail, right now we are outputting only the failed tests logs. With this, both pass and fail tests will get their logs outputted.
* If all tests fail, same logic will apply as today
* It all tests pass, we will log the output of all tests once this option is enabled via magellan.json

Reply:
By using `else if` it would choose only one condition, if you would like both sides logging another `if` should be a correct way, isn't it?

Reply:
Sorry my mistake, this is only for runs where all tests pass.

Reply:
I fixed the logic as well since I noticed analytics would stop firing in Magellan on CI